__tests__/Port.test.js

```diff
@@ -0,0 +1,38 @@
+const Port = require('../src/Port');
+
+describe('Port', () => {
+    describe('monitors ships in and leaving port', () => {
+        let plymouth;
+        let ship;
+        let titanic;
+        let queenMary;
+        beforeEach(() => {
+            plymouth = new Port('Plymouth');
+            ship = jest.fn();
+            titanic = jest.fn();
+            queenMary = jest.fn();
+    });
```

Comment:
Ending brackets of the beforeEach block should be at the same level as the start:
```
        beforeEach(() => {
            plymouth = new Port('Plymouth');
            ship = jest.fn();
            titanic = jest.fn();
            queenMary = jest.fn();
        });
```